src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -42,6 +49,17 @@ public EnvelopeProcessor(EnvelopeRepository envelopeRepository) {
             envelope.getZipFileName()
         );
 
-        return dbEnvelope.getScannableItems();
+        return dbEnvelope;
+    }
+
+    @Transactional
```

Comment:
Do we need this ?
> CRUD methods on repository instances are transactional by default. For reading operations the transaction configuration readOnly flag is set to true, all others are configured with a plain @Transactional so that default transaction configuration applies.

Reply:
Removed altogether with the other one